### PR TITLE
Model/refactor audio video

### PR DESCRIPTION
**Pull request by LimbertVargas**
Hello guys,
I correct some observations about documentation and name variables, please send me your observations,
Regards.

### CODE REVIEW

File: src/main/java/com/fundation/webservice/model/CriteriaAudio.java [hunk 1]
```diff
@@ -17,48 +17,60 @@
...
-    private int aRate;
+    private String audioCodec;
+    private int audioBit;
+    private int audioChannel;
+    private int audioRate;
 
+    /**
+     *
+     * Method getNewFormat receive the new format audio.
+     */
```
Comment (MadayAlcala) [inline src/main/java/com/fundation/webservice/model/CriteriaAudio.java:28]: Single line comment please.

File: src/main/java/com/fundation/webservice/model/CriteriaAudio.java [hunk 2]
```diff
@@ -17,48 +17,60 @@
...
     public String getNewFormat() {
         return newFormat;
     }
 
+    /**
+     *
+     * Method setNewFormat establish the new format audio that is define in getNewFormat
+     */
```
Comment (MadayAlcala) [inline src/main/java/com/fundation/webservice/model/CriteriaAudio.java:36]: Single line comment please.

File: src/main/java/com/fundation/webservice/model/CriteriaAudio.java [hunk 3]
```diff
@@ -17,48 +17,60 @@
...
     public void setNewFormat(String newFormat) {
         this.newFormat = newFormat;
     }
 
-    public String getaCodec() {
-        return aCodec;
+    /**
+     *
+     * Method getAudioCodec to work in the format audio required.
+     */
```
Comment (MadayAlcala) [inline src/main/java/com/fundation/webservice/model/CriteriaAudio.java:44]: Single line comment please.

File: src/main/java/com/fundation/webservice/model/CriteriaAudio.java [hunk 4]
```diff
@@ -17,48 +17,60 @@
...
+    public String getAudioCodec() {
+        return audioCodec;
     }
 
-    public void setaCodec(String aCodec) {
-        this.aCodec = aCodec;
+    public void setAudioCodec(String audioCodec) {
```
Comment (MadayAlcala) [inline src/main/java/com/fundation/webservice/model/CriteriaAudio.java:49]: Comments please

File: src/main/java/com/fundation/webservice/model/CriteriaAudio.java [hunk 5]
```diff
@@ -17,48 +17,60 @@
...
+        this.audioCodec = audioCodec;
     }
 
-    public int getaBit() {
-        return aBit;
+    public int getAudioBit() {
```
Comment (MadayAlcala) [inline src/main/java/com/fundation/webservice/model/CriteriaAudio.java:53]: comments please

File: src/main/java/com/fundation/webservice/model/CriteriaAudio.java [hunk 6]
```diff
@@ -17,48 +17,60 @@
...
+        return audioBit;
     }
 
-    public void setaBit(int aBit) {
-        this.aBit = aBit;
+    public void setAudioBit(int audioBit) {
```
Comment (MadayAlcala) [inline src/main/java/com/fundation/webservice/model/CriteriaAudio.java:57]: comments please

File: src/main/java/com/fundation/webservice/model/CriteriaAudio.java [hunk 7]
```diff
@@ -17,48 +17,60 @@
...
+        this.audioBit = audioBit;
     }
 
-    public int getaChannel() {
-        return aChannel;
+    public int getAudioChannel() {
```
Comment (MadayAlcala) [inline src/main/java/com/fundation/webservice/model/CriteriaAudio.java:61]: comments please

File: src/main/java/com/fundation/webservice/model/CriteriaAudio.java [hunk 8]
```diff
@@ -17,48 +17,60 @@
...
+        return audioChannel;
     }
 
-    public void setaChannel(int aChannel) {
-        this.aChannel = aChannel;
+    public void setAudioChannel(int audioChannel) {
```
Comment (MadayAlcala) [inline src/main/java/com/fundation/webservice/model/CriteriaAudio.java:65]: comments please

File: src/main/java/com/fundation/webservice/model/CriteriaAudio.java [hunk 9]
```diff
@@ -17,48 +17,60 @@
...
+        this.audioChannel = audioChannel;
     }
 
-    public int getaRate() {
-        return aRate;
+    public int getAudioRate() {
```
Comment (MadayAlcala) [inline src/main/java/com/fundation/webservice/model/CriteriaAudio.java:69]: comments please

File: src/main/java/com/fundation/webservice/model/CriteriaAudio.java [hunk 10]
```diff
@@ -17,48 +17,60 @@
...
+        return audioRate;
     }
 
-    public void setaRate(int aRate) {
-        this.aRate = aRate;
+    public void setAudioRate(int audioRate) {
```
Comment (MadayAlcala) [inline src/main/java/com/fundation/webservice/model/CriteriaAudio.java:73]: comments please

File: src/main/java/com/fundation/webservice/model/CriteriaAudio.java (hunk 1 above)
Comment (MadayAlcala) [reply src/main/java/com/fundation/webservice/model/CriteriaAudio.java:28]: The comment in third person please. You can review examples in the class "CriteriaThumbnailVideo.java".

File: src/main/java/com/fundation/webservice/controller/Controller.java [hunk 11]
```diff
@@ -135,10 +135,10 @@ public AudioResponse upload(@RequestParam("audio") MultipartFile file,
         criterion.setSrcPath("C:\\_pg\\tmp\\uploads\\" + fileName);
```
Comment (MadayAlcala) [inline src/main/java/com/fundation/webservice/controller/Controller.java:135]: The variable 'criterion' must be changed to 'criteria' in all classes and methods, as we left in group meeting.

File: src/main/java/com/fundation/webservice/controller/Controller.java [hunk 12]
```diff
@@ -135,10 +135,10 @@ public AudioResponse upload(@RequestParam("audio") MultipartFile file,
...
         criterion.setDestPath("C:\\_pg\\tmp\\conversions\\" + newName + extension);
```
Comment (jpsandovaln) [inline src/main/java/com/fundation/webservice/controller/Controller.java:136]: please use relative path for tmp folder until we have it on a config file

File: src/main/java/com/fundation/webservice/model/AudioConvert.java [hunk 13]
```diff
@@ -23,25 +23,32 @@
  * @version 1.0
  */
 public class AudioConvert implements IConvert{
-    private CriteriaAudio criterion;
+    private CriteriaAudio criteria;
 
-    public AudioConvert(CriteriaAudio criterion) {
-        this.criterion = criterion;
+    /**
+     *
```
Comment (jpsandovaln) [inline src/main/java/com/fundation/webservice/model/AudioConvert.java:29]: Add description please

File: src/main/java/com/fundation/webservice/controller/Controller.java (hunk 11 above)
Comment (LimbertVargas) [reply src/main/java/com/fundation/webservice/controller/Controller.java:135]: this class and methods are changed in the next branch and it is irrelevant to make changes now

File: src/main/java/com/fundation/webservice/model/CriteriaAudio.java (hunk 2 above)
Comment (LimbertVargas) [reply src/main/java/com/fundation/webservice/model/CriteriaAudio.java:36]: it is the name of the class for emphasizing the comment

File: src/main/java/com/fundation/webservice/model/CriteriaAudio.java (hunk 3 above)
Comment (LimbertVargas) [reply src/main/java/com/fundation/webservice/model/CriteriaAudio.java:44]: it is the name of the class for emphasizing the comment

File: src/main/java/com/fundation/webservice/model/CriteriaAudio.java (hunk 1 above)
Comment (LimbertVargas) [reply src/main/java/com/fundation/webservice/model/CriteriaAudio.java:28]: this documentation can review later please.

File: src/main/java/com/fundation/webservice/model/CriteriaAudio.java (hunk 4 above)
Comment (LimbertVargas) [reply src/main/java/com/fundation/webservice/model/CriteriaAudio.java:49]: this documentation can review later please.

File: src/main/java/com/fundation/webservice/model/CriteriaAudio.java (hunk 6 above)
Comment (LimbertVargas) [reply src/main/java/com/fundation/webservice/model/CriteriaAudio.java:57]: this documentation can review later please.

File: src/main/java/com/fundation/webservice/model/CriteriaAudio.java (hunk 7 above)
Comment (LimbertVargas) [reply src/main/java/com/fundation/webservice/model/CriteriaAudio.java:61]: this documentation can review later please.

File: src/main/java/com/fundation/webservice/model/CriteriaAudio.java (hunk 8 above)
Comment (LimbertVargas) [reply src/main/java/com/fundation/webservice/model/CriteriaAudio.java:65]: this documentation can review later please.

File: src/main/java/com/fundation/webservice/model/CriteriaAudio.java (hunk 10 above)
Comment (LimbertVargas) [reply src/main/java/com/fundation/webservice/model/CriteriaAudio.java:73]: this documentation can review later please.

**Pull request closed without merging**